ntbs-service/Pages/Search/Index.cshtml.cs

```diff
@@ -90,12 +90,13 @@ public async Task<IActionResult> OnGetAsync(int? pageIndex)
                 .FilterByTBService(SearchParameters.TBServiceCode)
                 .GetResult();
 
-            Tuple<IList<int>, int> notificationIdsAndCount = await searchService.UnionAndPaginateQueryables(filteredDrafts, filteredNonDrafts, PaginationParameters);
-            var notificationIds = notificationIdsAndCount.Item1;
+            Tuple<IList<int>, int> notificationIdsAndCount = await searchService.OrderAndPaginateQueryables(filteredDrafts, filteredNonDrafts, PaginationParameters);
+            var orderedNotificationIds = notificationIdsAndCount.Item1;

```

Comment:
Sorry I didn't think of it at the initial PR...
This could be a good use of C# 7's value tuple:
https://blogs.msdn.microsoft.com/mazhou/2017/05/26/c-7-series-part-1-value-tuples/

Reply:
Awesome - these are great, will change

Reply:
those *are* great. Liking c# more and more!